File: php/tests/VersionTest.php
<?php

declare(strict_types=1);

require __DIR__ . '/bootstrap.php';

use Itcrow\Uacryptex\Uacryptex;

$version = Uacryptex::libraryVersion();
if ($version !== Uacryptex::VERSION) {
    fwrite(STDERR, "expected version " . Uacryptex::VERSION . ", got {$version}\n");
    exit(1);
}

echo "OK: {$version}\n";

File: go/uacryptex/version_test.go
package uacryptex_test

import (
	"testing"

	"github.com/itcrow/uacryptex/uacryptex"
)

func TestLibraryVersion(t *testing.T) {
	v, err := uacryptex.LibraryVersion()
	if err != nil {
		t.Fatalf("LibraryVersion: %v", err)
	}
	if v != uacryptex.Version {
		t.Fatalf("got version %q, want %q", v, uacryptex.Version)
	}
}
